Content/home.css

```diff
@@ -85,6 +85,126 @@ section {
   gap: 2rem;
 }
 
+/* Form Container */
+.form-container {
+  width: 465px;
+  /* height: 314px; */
+  height: 100%;
+  margin: 0px auto;
+  background: white;
+  border-radius: 8px;
+  padding: 20px;
+  /* box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); */
+  border: 1px solid #e6e6e6;
+}
+
+/* Labels and Inputs */
+label {
+  display: block;
+  font-size: 14px;
+  font-weight: bold;
+  margin-bottom: 5px; /* Spacing between label and input */
+  color: #333;
```

Comment:
Replace hardcoded colors (#333, #003366, etc.) with CSS variables to promote consistency and easy updates.

Example:
```css
:root {
    --text-color: #333;
}
```